src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/layout/index.ts

```diff
@@ -18,6 +18,7 @@ export interface ILayoutGroup extends ILayoutEntry {
   textResourceBindings?: ITextResourceBindings;
   tableHeaders?: string[];
   edit?: IGroupEditProperties;
+  openDefault?: boolean;
```

Comment:
I suggest renaming this to `openByDefault` and move this prop to the edit-object.

So that we can set up the component with
```
{
  "type": "group"
   ...
   "edit": {
     "openByDefault": true
   }
}
```